anta/tests/cvx.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+"""Module related to the CVX tests."""
+
+# Mypy does not understand AntaTest.Input typing
+# mypy: disable-error-code=attr-defined
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, ClassVar
+
+from anta.models import AntaCommand, AntaTest
+
+if TYPE_CHECKING:
+    from anta.models import AntaTemplate
+
+
+class VerifyMcsClientMounts(AntaTest):
+    """Verifies the MCS Clients mount status.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the MCS mount status on MCS Clients are mountStateMountComplete.
+    * Failure: The test will fail even if one switch's MCS client mount status is not  mountStateMountComplete.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.cvx:
+    - VerifyMcsClientMounts:
+    ```
+    """
+
+    name = "VerifyMcsClientMounts"
+    description = "Verify if all MCS client mounts are in mountStateMountComplete"
```

Comment:
We merged a PR that makes these optional - name will be the class name by default and description the first line of the docstring!
```suggestion
```